script.js

```diff
@@ -61,9 +64,10 @@ modalClose.addEventListener("click", function(event) {
     closeModal();
 });
 
-searchButton.addEventListener(`click`, event => {
+searchDistance.addEventListener(`click`, event => {
+    clearResults ()
     fetch(
-            `https://developers.zomato.com/api/v2.1/search?lat=${userLatitude}&lon=${userLongitude}&apikey=969dccb114a560b6d4df35b25a8e6418`
+            `https://developers.zomato.com/api/v2.1/search?lat=${userLatitude}&lon=${userLongitude}&apikey=969dccb114a560b6d4df35b25a8e6418&sort=real-distance`
         )
         .then(repsonse => repsonse.json())
```

Comment:
there is a typo here - should be "response" and not "repsonse"

Reply:
Ive made the changes to the spelling mistakes